src/views/v2/month/calendar-body/day/multiday-events/multiday-event/hidden/link/featured.php

```diff
@@ -23,7 +23,9 @@
 }
 ?>
 <em
-	class="tribe-events-calendar-month__multiday-event-hidden-featured-icon tribe-common-svgicon tribe-common-svgicon--featured"
+	class="tribe-events-calendar-month__multiday-event-hidden-featured-icon"
 	aria-label="<?php esc_attr_e( 'Featured', 'the-events-calendar' ); ?>"
 	title="<?php esc_attr_e( 'Featured', 'the-events-calendar' ); ?>"
-></em>
+>
+	<?php $this->template('components/icons/featured', [ 'classes' => ['tribe-events-calendar-month__multiday-event-hidden-featured-icon-svg'] ] ) ?>
```

Comment:
* :no_entry_sign: Inline PHP statement must end with a semicolon
* :no_entry_sign: Expected 1 spaces after opening parenthesis; 0 found
* :no_entry_sign: There must be a single space after the first parenthesis
* :no_entry_sign: Missing space after array opener.
* :no_entry_sign: Missing space after array opener.
* :no_entry_sign: Missing space before array closer.
* :no_entry_sign: Missing space before array closer.

<sub>hash: fc7b713393acf5879d220b6128328a11</sub>